lib/tasks/ckeditor.rake

```diff
@@ -1,13 +1,26 @@
 require 'fileutils'
 
-desc "Create nondigest versions of all ckeditor digest assets"
-task "assets:precompile" do
-  fingerprint = /\-[0-9a-f]{32}\./
-  for file in Dir["public/assets/ckeditor/**/*"]
-    next unless file =~ fingerprint
-    nondigest = file.sub fingerprint, '.'
-    if !File.exist?(nondigest) or File.mtime(file) > File.mtime(nondigest)
-      FileUtils.cp file, nondigest, verbose: true, preserve: true
+namespace "ckeditor" do
+  desc "Create nondigest versions of all ckeditor digest assets"
+  task "nondigest" => [:environment] do
+    fingerprint = /\-[0-9a-f]{32}\./
```

Comment:
maybe let's stick to `{32,64}` so it will be backward compatible, what do you say?